tests/_serialization/test_container_properties.py

```diff
@@ -42,3 +42,19 @@ def test_mapping_property(key_type, value_type, key_value, value_value, key_seri
     serialized = {key_serialized: value_serialized}
     assert prop.deserialize(serialized) == value
     assert prop.serialize(value) == serialized
+
+@pytest.mark.parametrize('key_type,key_value,key_serialized', VALID_SERIALIZATIONS)
+@pytest.mark.parametrize('value_type,value_value,value_serialized', VALID_SERIALIZATIONS)
+def test_mapping_property_list_of_pairs(key_type, value_type, key_value, value_value, key_serialized, value_serialized):
+    prop = properties.Mapping(key_type, value_type, list_of_pairs = True)
+    value = {key_value: value_value}
+    serialized = [(key_serialized, value_serialized),]
+    assert prop.deserialize(serialized) == value
+    assert prop.serialize(value) == serialized
+
+def test_mapping_property_list_of_pairs_multiple():
```

Comment:
going to actually merge these two tests with something like `{key_value: value_value for _ in range(5)}` instead

Reply:
jk, not that, because keys need to be unique in dictionaries :derp: